content/news/2023-10-13-EGD-2023-the-wrapup/index.md

```diff
@@ -0,0 +1,141 @@
+
+Like for the [EGD 2022](/events/2022-10-egd/), effectively we had a triple event also in 2023, joining forces again with [ELIXIR](https://elixir-europe.org) and [EuroScienceGateway](/projects/esg/) (ESG), a project from the [EOSC space](https://eosc.eu), which effectivly is driven by the European Galaxy community. 
+
+### ELIXIR Galaxy Community F2F Meeting
+
+[ELIXIR's Galaxy Community](https://elixir-europe.org/communities/galaxy) finally could have its (first) [face-to-face meeting](https://elixir-europe.org/events/elixir-galaxy-community-meeting-0), after in the beginning of the year the strong wish arose to meet in person. So, what would have been a better place for a F2F than the EGD, meeting also non-ELIXIR people, enjoy more talks, and present achievements to the larger public? 
+
+Consequently, we had a dedicated ELIXIR session on Day 1, [introducing the community to the public](https://galaxyproject.org/events/2023-10-egd/egd/ELIXIR-presentation_2023_10_Galaxy-Community-F2F_EGD_presentation.pdf), but also providing a broad overview on community-derived topics: [Fair Data Management training](https://docs.google.com/presentation/d/1SWO73pjufZI44KjkHg_EUzjMhcgOAeHy/edit#slide=id.g287f34dccc2_0_1098), [mass spectrometry](https://docs.google.com/presentation/d/1FNj_qEPQ2bDHMqNp-qjj1iEnqeArEIm4/edit#slide=id.p1), [connecting OMERO to Galaxy for image analysis](https://galaxyproject.org/events/2023-10-egd/egd/Lucille_Delisle_galaxy_omero.pdf) and [genome annotations on really large scale](https://docs.google.com/presentation/d/1BwZ7W_tg2kbVwIKRsN4abOAtuQJCGIcOkr1tGrO5sKQ/edit#slide=id.g742e3e7cd_1_16). A visual highlight was the presentation officially closing our community's participation in the [ELIXIR staff exchange programme](https://elixir-europe.org/internal-projects/staff-exchange-programme), which enabled 14 members to visit the [GCC 2023 in Brisbane, and beyond](https://elixir-europe.org/news/australia_visit_2023) integrate deeper with Australian colleagues and organizations.
+
+Speaking of funding we like to emphasize ELIXIR's ([again!](/events/2022-10-egd/#supporters)) great role in enabling us again to carry out the EGD. We thank both Hub and DE node for their generous funding of the _overall_ event and supportive way in organizing!
+
+### EuroScienceGateway's first General Assembly (GA) 
+
+The sub-meeting mostly set on Day 3 in parallel with the CoFest marked the project's turnover from year 1 to year 2, and entering the finalization phase for our first report (which is why we had a dedicated BoF). Thus, we had a wrap-up of our achievements, and could note that we are well on track having ESG's proposal in mind. 
```

Comment:
Day 3 = day 3

"could note that we are well on track having ESG's proposal in mind. "
 planning of year 2 and proposal alignment/expectation was also a topic - even more important than talking about what was done